templates/sso-cd-mysql-persistent.json

```diff
@@ -693,6 +705,12 @@
                                 "secret": {
                                     "secretName": "${SSO_TRUSTSTORE_SECRET}"
                                 }
+                            },
+                            {
+                                "name": "sso-vault-volume",
+                                "secret": {
+                                    "secretName": "${SSO_VAULT_SECRET}"
```

Comment:
@martin-kanis , do you know what happens if `SSO_VAULT_SECRET` is unset? Will be the volume mounted anyway? If yes this will result in adding vault provider to standalone.xml always. Which is something we probably don't want.